index.html

```diff
@@ -223,7 +258,8 @@ <h3> Normative Specifications </h3>
             </dd>
             <dd><b>Adopted Working Draft:</b> Timed
               Text Markup Language 2 (TTML2), <a href="https://www.w3.org/TR/2018/CR-ttml2-20180313/">Candidate Recommendation, 13 March
-              2018.</a>
+              2018</a>.
+
```

Comment:
We should update this to refer to a new 2nd edition WD (or ED).

Reply:
The deliverables do need updating, but not in this pull request, which is tackling modularisation for TTML3 only. I'll tackle this elsewhere - see https://github.com/w3c/charter-timed-text/pull/42#pullrequestreview-215521606